jerry-core/lit/lit-magic-strings.h

```diff
@@ -34,6 +34,11 @@ typedef enum
 #include "lit-magic-strings.inc.h"
 #undef LIT_MAGIC_STRING_DEF
 #undef LIT_MAGIC_STRING_FIRST_STRING_WITH_SIZE
+  LIT_REAL_MAGIC_STRING__COUNT, /**< number of real magic strings */
```

Comment:
Renamed to LIT_NON_INTERNAL_MAGIC_STRING__COUNT and INTERNAL prefix is added to the new strings as well.